governance.md

```diff
@@ -1,14 +1,32 @@
-# CDEvents Bootstrap Governance
+# CDEvents Governance
 
-The initial bootstrap committee will consist of 5 individuals who are core stakeholders and/or contributors.
+The CDEvents Governance committee is the governing body of the CDEvents open source project. It's an elected group that represents the contributors to the project, and has an oversight on governance and technical matters.
 
-Members are (in alphabetical order):
 
-* Emil Bäckmark, [@e-backmark-ericsson](https://github.com/e-backmark-ericsson), Ericsson
-* Andrea Frittoli, [@afrittoli](https://github.com/afrittoli), IBM
-* Mattias Linnér, [@m-linner-ericsson](https://github.com/m-linner-ericsson), Ericsson
-* Erik Sternerson [@erkist](https://github.com/erkist), doWhile
-* Steve Taylor [@sbtaylor15](https://github.com/sbtaylor15), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
+- [CDEvents Governance Committee](#CDEvents-governance-committee)
+- [Governance Facilitator Role (optional)](#governance-facilitator-role-optional)
+- [Maximum Representation](#maximum-representation)
+- [Committee Responsibilities and Deliverables](#committee-responsibilities-and-deliverables)
+- [Governance Meetings and Decision-Making Process](#governance-meetings-and-decision-making-process)
+- [Elections](#elections)
+- [Permissions and access](#permissions-and-access)
+- [Governing board leave policy](#governing-board-leave-policy)
+
+## CDEvents Governance Committee
+
+The CDEvents governance committee consists of five elected individuals. The five seats are held for two year terms, staggered by one year: every year either two or three of the seats are up for election.
+
+### Current members (Bootstrap)
+
+| Full Name         | Company    | GitHub                                      | Slack                                                         | From | Until    |
+|-------------------|:----------:|---------------------------------------------|---------------------------------------------------------------|------------|----------|
+| Emil Bäckmark     | Ericsson   | [@e-backmark-ericsson](https://github.com/e-backmark-ericsson) | [@Emil Bäckmark](https://cdeliveryfdn.slack.com/team/USR6E330D) | Oct 2021 | 1 year after first election |
+| Andrea Frittoli   | IBM        | [@afrittoli](https://github.com/afrittoli)  | [@Andrea Frittoli](https://cdeliveryfdn.slack.com/team/URNB1NTGR)   | Oct 2021 | 1 year after first |
+| Mattias Linnér    | Ericsson   | [@m-linner-ericsson](https://github.com/m-linner-ericsson) | [@Mattias Linnér](https://cdeliveryfdn.slack.com/team/U01AHR341T8) | Oct 2021 | First Election |
```

Comment:
You have been part of the bootstrap governing board since the beginning so I maintained the list, but if you'd prefer to renounce please let me know, I will remove your name.

In practice, Emil and myself have acted as bootstrap governing board, which is why I suggested we stay on the GB for 1 year for continuity and then our seats will be up for election (we may still be candidate and elected)

Reply:
If you need me I can stay. It was just that I have not been active so it first sounded strange for me to have my name there.

Reply:
Aha, this was about the former Governing board members.

Reply:
Thank you for the offer. This is simply stating the current situation, no change. Once we have elections your current seat on the bootstrap GB will be up for election.